### PR TITLE
Fix: Typography

### LINKED ISSUE #141: Fix: Typography font sizes
I think the typography font sizes are too small to be able to read properly. Also, there is no regular increase in font size from h6 to h1, making it difficult for developers to follow a pattern.
You can check here the typography configuration: [https://github.com/codeforcauseorg/edu-client/blob/main/src/theme/typography.js](url)

### PR DESCRIPTION
**Pull request by faisaljamil25**
**Issue Number**

fixes #141 

**Describe the changes you've made**

Typography configuration has changed and font sizes are increased. Also fixed font sizes in some files.

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

`NA`

**Checklist**

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by faisaljamil25** [comment]
@Abhishek-kumar09 please review this and let me know any changes.

**Review by Abhijay007 — COMMENTED** [review]
typo

@faisaljamil25 LGTM 👍

File: src/theme/typography.js [hunk 1]
```diff
@@ -4,49 +4,59 @@ export default {
 
   h1: {
     fontWeight: 700,
-    fontSize: 35,
+    fontSize: 50,
     letterSpacing: "-0.24px",
```
Comment (Abhishek-kumar09) [inline src/theme/typography.js:8]: The sizes are too big according to me. 
This is the the typography you can look to https://github.com/codeforcauseorg/codeforcause.org/blob/HEAD/src/theme/typography.js

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
The sizes are too big.
Also some fontweights are also changed.
e.g: 
[expected](https://user-images.githubusercontent.com/48255244/112674379-dfb96680-8e8b-11eb-9baa-ac97367726cb.png)
[Actual](https://user-images.githubusercontent.com/48255244/112674566-198a6d00-8e8c-11eb-8cad-81f5a82031ee.png)

**Comment by Abhishek-kumar09** [comment]
Please also share screenshots of pages

**Comment by faisaljamil25** [comment]
Screenshot 2021-03-27 005642
Screenshot 2021-03-27 005738
Screenshot 2021-03-27 005821
Screenshot 2021-03-27 005623
Screenshot 2021-03-27 005642

do I need to reduce the font size more?

File: src/theme/typography.js [hunk 2]
```diff
@@ -1,52 +1,62 @@
...
-    fontWeight: 700,
-    fontSize: 12,
+    fontWeight: 600,
+    fontSize: 20,
     letterSpacing: "-0.06px",
   },
   h4: {
-    fontWeight: 600,
-    fontSize: 20,
+    fontWeight: 500,
```
Comment (Abhishek-kumar09) [inline src/theme/typography.js:21]: make it 600

File: src/theme/typography.js [hunk 3]
```diff
@@ -1,52 +1,62 @@
...
+    fontSize: 18,
     letterSpacing: "-0.06px",
   },
   h5: {
-    fontWeight: 600,
+    fontWeight: 500,
```
Comment (Abhishek-kumar09) [inline src/theme/typography.js:26]: 600

File: src/theme/typography.js [hunk 4]
```diff
@@ -1,52 +1,62 @@
...
   h2: {
     fontWeight: 600,
-    fontSize: 20,
+    fontSize: 28,
     letterSpacing: "-0.24px",
   },
   h3: {
-    fontWeight: 700,
-    fontSize: 12,
+    fontWeight: 600,
```
Comment (Abhishek-kumar09) [inline src/theme/typography.js:16]: 700

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Comment by Abhishek-kumar09** [comment]
Make this a bit smaller and thicker 
image

**Comment by Abhishek-kumar09** [comment]
The less there will be changes in current view of app, the better it would be

**Comment by faisaljamil25** [comment]
Screenshot 2021-03-27 033204
Does this look good?

**Comment by Abhishek-kumar09** [comment]
Hi @faisaljamil25 
This is the figma design: https://www.figma.com/file/3QxrkclcjQFa2dplfWrQ8v/CFC-Application-(Copy)

Please ensure the changes align with the design.

**Comment by faisaljamil25** [comment]
> Hi @faisaljamil25
> This is the figma design: https://www.figma.com/file/3QxrkclcjQFa2dplfWrQ8v/CFC-Application-(Copy)
> 
> Please ensure the changes align with the design.

@Abhishek-kumar09 I have almost made the typography and font sizes identical or near to the one that you have provided and also changed some places where font sizes and font weights are hardcoded but still, a lot of places are left to be redefined and it would take a whole lot of time to correct each place to make as identical with the design (say if I increase font-weight of h3 to make align at one place it may be possible that it breaks font style at some other place and then I have to go to those individual places to change their variants and who knows correcting that place font style may lead to change in font-weight of some other variant and the loop continues). So, I think it would be better if we define a common typography configuration and then change the font style while working on those files themselves. What do you think?

**Comment by faisaljamil25** [comment]
@Abhishek-kumar09 have a look at this:
Screenshot 2021-03-28 123146
Screenshot 2021-03-28 123223

**Comment by Abhishek-kumar09** [comment]
Looks good, Thanks @faisaljamil25

**Pull request merged**